src/Grocery/AddGrocery.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react';
+import Meteor from 'react-native-meteor';
+
+import { colors } from '../../config/styles';
+
+import { TextInput, StyleSheet, SafeAreaView } from 'react-native';
+import { Button } from 'react-native-elements';
+
+export default class AddGrocery extends Component {
+	
+	constructor(props) {
+		super(props);
+		
+		// TODO allow users to add an amount as well
+		this.state = {
+			name: ''
+		}
+	}
+    
+	addGrocery() {
+		if (this.state.name.length === 0) {
+			return
+		}
+
+		Meteor.call('groceries.insert', this.state.name, null, (err, groceryId) => {
+			if (err) {
+				// TODO do something
+			}
+
+			Meteor.call('grocerylists.addItem', this.props.navigation.state.params.listId, groceryId);
+			this.props.navigation.goBack();
+		});
+	}
+
+	static navigationOptions({ navigation }) {
+		return {
+			headerTitle: 'New Grocery Item',
+			headerRight: (
+				<Button 
+					title='Done'
+					onPress={() => this.addGrocery()}
+					backgroundColor={colors.background}/>
+			),
+			headerLeft: (
+				<Button 
+					title="Cancel"
```

Comment:
That's true. I just got used to using double quotes in case I need to use single quotes in the text (e.g. "don't" or "can't") but don't have a strong preference

Reply:
Yeah double is probably better then